Mage.Sets/src/mage/cards/c/CrystalSlipper.java

```diff
@@ -27,10 +27,8 @@ public CrystalSlipper(UUID ownerId, CardSetInfo setInfo) {
         this.subtype.add(SubType.EQUIPMENT);
 
         // Equipped creature gets +1/+0 and has haste.
-        Ability ability = new SimpleStaticAbility(new BoostEquippedEffect(1, 0));
-        ability.addEffect(new GainAbilityAttachedEffect(
-                HasteAbility.getInstance(), AttachmentType.EQUIPMENT).setText("and has haste")
-        );
```

Comment:
I changed this to fix the text, I just forgot to add `this.addAbility(ability);`

Reply:
Changed it back and added the missing part.

Reply:
Great, thanks for your contributions